src/Native/Windows/CMakeLists.txt

```diff
@@ -113,10 +111,6 @@ set(__LinkArgs "${__LinkArgs} /PDBCOMPRESS")            #shrink pdb size
 set(__LinkArgs "${__LinkArgs} /DEBUG")
 set(__LinkArgs "${__LinkArgs} /IGNORE:4197,4013,4254,4070,4221")
 
-if ($ENV{__BuildArch} STREQUAL "x86" OR $ENV{__BuildArch} STREQUAL "x86_64" )
```

Comment:
What's the reason for this change?

Reply:
Every x86 build in CoreFX is getting this warning:

```
LINK : warning LNK4010: invalid subsystem version number 6.00; default subsystem version assumed
```

then i read https://github.com/dotnet/coreclr/pull/8211#discussion_r88793983 and thought it is not needed.

Reply:
Oh boy, I wished I knew where it is implemented in C# where it would generate docs for internal / private functions with public ones as well. I thought including doc summaries for internal / private functions would be excloded from generated documentation xml's and used only by Core (in the case of my project) developers. Which would be a great change as well.

Reply:
ok